recipes/tools/readme.md

```diff
@@ -25,24 +25,15 @@ If you'd like an UI with info about your swarm:
         portainer/portainer \
         -H unix:///var/run/docker.sock
 
-- ##### [docker-swarm-visualizer](https://github.com/dockersamples/docker-swarm-visualizer)
-If you'd like to have a basic view of the distribution of containers in your swarm cluster, add the following service to your docker-compose.yml file.
+- ### [docker-swarm-visualizer](https://github.com/dockersamples/docker-swarm-visualizer)
+If you'd like to have a basic view of the distribution of containers in your swarm cluster, you can use the ```visualzer.yml``` file provided in this folder.
 
-        viz:
-          image: dockersamples/visualizer
-          volumes:
-            - /var/run/docker.sock:/var/run/docker.sock
-          deploy:
-            placement:
-              constraints: [node.role == manager]
-          ports:
-            - "8282:8080"
-          networks:
-            - backend
-            - frontend
+    docker stack deploy -c visualizer.yml vis
+    
+- ### [postman](https://www.getpostman.com/)
+A well-known tool for experimenting with APIs. Do you want to try the curl-based examples of the recipes from Postman? Import the ```postman_collection.json``` available in this folder and make your tests easier. Note: this collection is work in progress, feel free to [contribute](../contributing.md)!
 
-
-## Docs-related
+## Writing Docs?
 
```

Comment:
i would add a reference to the architecture diagram project

Reply:
good idea! done